CHANGELOG.md

```diff
@@ -53,6 +53,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 * [#2804](https://github.com/osmosis-labs/osmosis/pull/2804) Improve error messages when parsing pool assets.
 * [#3214](https://github.com/osmosis-labs/osmosis/pull/3214) Add basic CLI query support for TWAP.
 
+## v12.3.0
```

Comment:
Please move it above "v12.2.1" and then LGTM

Reply:
Ohh sorry I miss this

Reply:
Np, thanks